.nvmrc

```diff
@@ -1 +1 @@
-18.16
```

Comment:
_suggestion_: revert this change, because it does not belong to the ci version update or create a separate commit for it (chore: updated development version).

Reply:
done